src/openms/source/ANALYSIS/RNPXL/ModifiedPeptideGenerator.cpp

```diff
@@ -33,32 +33,79 @@
 // --------------------------------------------------------------------------
 
 #include <OpenMS/ANALYSIS/RNPXL/ModifiedPeptideGenerator.h>
-
+#include <OpenMS/CHEMISTRY/ResidueDB.h>
+#include <OpenMS/CHEMISTRY/ModificationsDB.h>
 using std::vector;
 using std::map;
 
 namespace OpenMS
 {
 
+ // static
+ ModifiedPeptideGenerator::MapToResidueType ModifiedPeptideGenerator::getModifications(const StringList& modNames)
+ {   
+   vector<const ResidueModification*> modifications;
+  
+   for (String modification : modNames)
+   {
+     const ResidueModification* rm;
+
+     // TODO: check if this is still required
```

Comment:
is it required?

Reply:
I changed the relevant part so it is not required anymore